1-exercises/F-for-of-loop/exercise.js

```diff
@@ -11,6 +11,13 @@ let tubeStations = [
     "Tottenham Court Road"
 ];
 
+for (let i=0; i<tubeStations.length; i++){
```

Comment:
Line 14 and 21 are for loops but not for-of loops as they do not contain the keyword "of". Could you replace these for loops with for-of loops. Examples of for-of loops can be found on MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of